pynet/datasets/hcp.py

```diff
@@ -0,0 +1,104 @@
+import os
+from collections import namedtuple
+
+import boto3
+import nibabel as nib
+import numpy as np
+import pandas as pd
+
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path"])
+
+
+def nii_to_np_arr(filename, low=False, binary=False):
+    img = nib.load(filename)
+    img_data = img.get_data()
+    img_data = np.append(img_data[2:-2, :, 2:-2], np.zeros((256, 9, 256)), axis=1)
+    if low:
+        img_data = img_data.reshape(32, 8, 40, 8, 32, 8).mean(axis=(1, 3, 5))
+
+    if binary:
+        img_data = 1 * (img_data > img_data.mean())
+    return img_data
+
+
+def extract_hcp_data(datasetdir, data, masks, s3, bucket, subject, label, low, metadata):
+    if label == 'T1w':
+        obj = s3.Object(bucket, subject + label + '/T1w_acpc_dc.nii.gz')
+    else:
+        obj = s3.Object(bucket, subject + label + '/T1w.nii.gz')
+
+    if not os.path.isdir(datasetdir + subject + label):
```

Comment:
Please use os.path.join() in order to be linux and windows compliant, ie. don't hardcode the path separator /.
Take this remark into account for the whole code.